src/test/java/smile/identity/core/WebApiTest.java

```diff
@@ -3,12 +3,53 @@
  */
 package smile.identity.core;
 
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.junit.Before;
 import org.junit.Test;
-import static org.junit.Assert.*;
+
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
 
 public class WebApiTest {
-    @Test public void testSomeLibraryMethod() {
-        // WebApi classUnderTest = new WebApi();
-        // assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+	
+	private static final int PORT = 32100; //Random port number, arbitrarily chosen
```

Comment:
The indentation in this file is inconsistent starting on this line and through the rest of the file. I recommend cleaning this up.

Reply:
Quite weird... 🤔